client/rpc/manual-xcm/src/lib.rs

```diff
@@ -43,30 +45,69 @@ pub trait ManualXcmApi {
 	/// The method accepts a sending paraId and a bytearray representing an arbitrary message as
 	/// parameters. If you provide an emtpy byte array, then a default message representing a
 	/// transfer of the sending paraId's native token will be injected.
-	#[rpc(name = "xcm_injectHrmpMessage")]
-	fn inject_hrmp_message(
-		&self,
-		sender: ParaId,
-		message: Vec<u8>,
-	) -> BoxFuture<'static, RpcResult<()>>;
+	#[method(name = "xcm_injectHrmpMessage")]
```

Comment:
Remove prefix `xcm_` since we use namespace ?

Reply:
Good catch, thank you :)